### PR TITLE
fix(kubelet): register ImageGarbageCollectedTotal metrics

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind cleanup
/kind bug

#### What this PR does / why we need it:
- register ImageGarbageCollectedTotal metrics, because it haven't registered before

In this PR #https://github.com/kubernetes/kubernetes/pull/123345, this metrics does not seem to be registered. 
#### Which issue(s) this PR fixes:

Fixes # None

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by haircommander** [comment]
whoops, thanks @googs1025 

/lgtm

**Comment by googs1025** [comment]
/cc @SergeyKanzhelev 
Can you help approve? thanks!

**Comment by googs1025** [comment]
@SergeyKanzhelev /PTAL thanks

File: pkg/kubelet/metrics/metrics.go [hunk 1]
```diff
@@ -932,6 +932,7 @@ func Register(collectors ...metrics.StableCollector) {
 		legacyregistry.MustRegister(PodStartSLIDuration)
 		legacyregistry.MustRegister(PodStartTotalDuration)
 		legacyregistry.MustRegister(ImagePullDuration)
+		legacyregistry.MustRegister(ImageGarbageCollectedTotal)
```
Comment (bart0sh) [inline pkg/kubelet/metrics/metrics.go:935]: /lgtm

**Comment by bart0sh** [comment]
/assign @SergeyKanzhelev @mrunalp 
for approval

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

**Pull request merged**